### PR TITLE
Refactor app insights telemetry to common helper class.  Add additional global telemetry to help with troubleshooting.

### PR DESCRIPTION
**Pull request by mazudo**
Refactoring telemetry to shared common helper class.  Added additional global telemetry data to troubleshoot devices internally at Microsoft.  External data remains protected with MS_INTERNAL_ONLY

### CODE REVIEW

File: SecuritySystemUWP/SecuritySystemUWP/TelemetryHelper.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+        /// <summary>
+        /// Log and track custom app insights event with global common properities
+        /// See https://azure.microsoft.com/en-us/documentation/articles/app-insights-api-custom-events-metrics/#api-summary
+        /// </summary>
+        /// <param name="eventName"></param>
+        internal static void TrackEvent(string eventName)
+        {
+            Dictionary<string, string> properties = new Dictionary<string, string>();
+            // call helper method to add common properties
```
Comment (mwmckee) [inline SecuritySystemUWP/SecuritySystemUWP/TelemetryHelper.cs]: Did you intend to actually make the call to the "addCommonProperties" method here? This comment is in a few places

Comment (mazudo) [inline SecuritySystemUWP/SecuritySystemUWP/TelemetryHelper.cs]: No, this actually calls the overloaded internal method in this same class, which in turn is already calling AddCommonProperties.  I'll update the comments to make this clearer.

**Comment by mwmckee** [comment]
signoff

**Comment by mazudo** [comment]
Slavi signed off as well via email.  Merging it in now.

**Pull request merged**